### PR TITLE
Rip out more dead analytics code

### PR DESCRIPTION
**Pull request by KrauseFx**
The code was unused, and we have no plans on adding this kind of analytics again

### CODE REVIEW

**Comment by fastlane-bot-helper** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by KrauseFx** [comment]
not big

File: fastlane/lib/fastlane/fast_file.rb [hunk 1]
```diff
@@ -201,9 +201,7 @@ def fastlane_require(gem_name)
     end
 
     def generated_fastfile_id(id)
-      # This value helps us track success/failure metrics for Fastfiles we
-      # generate as part of an automated process.
-      ENV['GENERATED_FASTFILE_ID'] = id
+      UI.important("The `generated_fastfile_id` action was deprected, you can remove the line from your Fastfile")
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/fast_file.rb]: ^

**Comment by KrauseFx** [comment]
big if true

File: fastlane/lib/fastlane/fast_file.rb (hunk 1 above)
Comment (janpio) [inline fastlane/lib/fastlane/fast_file.rb]:
```suggestion
      UI.important("The `generated_fastfile_id` action was deprecated, you can remove the line from your `Fastfile`")
```

**Review by joshdholtz — COMMENTED** [review]
Looks good to me after taking @janpio's feedback 👍

**Review by powerivq — APPROVED** [review]

**Pull request merged**